src/content/basics/multiaz/index.md

```diff
@@ -0,0 +1,56 @@
++++
+title = "Clusters Over Multiple Availability Zones"
+description = "By default, cluster get started within a single availability zone. But you can define how many availability zones a cluster should have to get higher a availability for your clusters."
+date = "2018-11-22"
+weight = 50
+type = "page"
+categories = ["basics"]
++++
+
+# Clusters Over Multiple Availability Zones
+
+*This is currently only available with Giant Swarm 6.1.0 on AWS*
+
+With Giant Swarm you can easily launch cluster across multiple availability zones (AZs). This will lower the risk that your cluster will get unavailable while there is an incident with your cloud providers or your datacenters infrastructure.
+
+## What availability zones are good for {#benefits}
+
+Both on AWS and Azure, support AZs within their regions. These zones still have good interconnectivity but are separated from each other to isolate failures within one zone from the others. So it is more likely that one AZ goes down than the whole region. If your clusters are running within multiple AZs then Kubernetes can easily shift your workloads from one AZ into the other.
+
```

Comment:
Shall we mention Azure if it is not yet implemented?

Reply:
Yes I'd keep it. The interface (gsctl/happa) will contain this for Azure too. It is just that the current release only supports one AZ.